packages/express-backend/src/persistence/mongoose-connection.js

```diff
@@ -88,6 +88,30 @@ async function createItem(item, uid) {
   }
 }
 
+async function addItemToUserBought(item, uid) {
+  try {
+    if (Object.keys(item).includes('userId')) {
+      throw new Error('Item passed to createItem must not include a userId')
+    }
+
+    const userId = new mongoose.Types.ObjectId(uid)
+
+    item.seller_id = userId
+
+    const itemToAdd = new Item(item)
+    const itemCreator = await findUserById(userId)
+    const savedItem = await itemToAdd.save()
```

Comment:
This looks like it creates a whole new item in the database. When an item is sold, shouldn't the existing item just be updated with a `buyer_id`?